nginx/ng/map.jinja

```diff
@@ -130,7 +130,7 @@
         'config': {
             'worker_processes': 'auto',
             'events': {
-                'worker_connections': 768,
+                'worker_connections': 1024,
```

Comment:
@garrettw And ask requested on Slack, a link to the upstream reference for this.